config/vufind/config.ini

```diff
@@ -163,6 +163,10 @@ allowSavedSearches = true
 ; still require Javascript; this simply improves compatibility for certain
 ; features (such as display of hierarchies).
 nonJavascriptSupportEnabled = false
+; Only display x number of subjects in the bib display on the full record by default.
+; Additional subjects are hidden by default and expandable via a "more..." button.
+; Enable by setting to a number, e.g. 3, to display 3 subjects and hide the rest.
+subjectLimit = false
```

Comment:
Would it make more sense to associate this with the search object (i.e. put in searches.ini or other equivalent configurations)? I think the answer is "no" because it is more likely that this would be desired globally -- but just playing devil's advocate and asking whether there will ever be a situation where we'll want different limits for different search backends.

Reply:
Interesting idea, I certainly wouldn't have thought of this. I think you're correct in that technically it's possible we might want it on some things and not others. It solves a problem for us in the immediate case, however, when searching another data type (e.g. something without availability or less metadata), it might not be needed. I'm still inclined to agree with your, "no" assessment for the following reasons:

1. Even if long subject listings caused a problem on one search backend and not another, I don't think it would harm the other and we might not care so much to configure them differently.
2. Consistency of UI.

But I don't feel strongly about this. We could still configure it the same if it were in `searches.ini` and it's easy enough to change. I'll defer to your judgement.

Reply:
I'm happy to leave it as it is for now, since it is more straightforward. We can always add another layer later if a compelling need arises.